test/serve-integration.js

```diff
@@ -6,7 +6,7 @@ process.on('SIGINT', async() => {
   if (stopping) return;
   stopping = true;
   console.log('shutting down');
-  await stopDocker();
+  // await stopDocker();
```

Comment:
code above looks good.  Not clear on this test case -- is it fully integrated so that we are just adding new test cases to all of the existing test cases?